content/docs/command-reference/status.md

```diff
@@ -192,9 +213,8 @@ matrix-train.p.dvc
             modified:  code/featurization.py
 ```
 
-If the `dvc status` command is limited to a target that had no changes, result
-shows no changes. By adding `--with-deps` the change will be found, so long as
-the change is in a preceding stage.
+If the `dvc status` command is limited to a target that had no changes, but by
+adding `--with-deps`, any upstream change will be found (in a preceding stage).
```

Comment:
upstream can be confusing with Git terminology

Reply:
OK. Updated in 8e7f63a7.